collect_app/src/main/java/org/odk/collect/android/utilities/ScreenName.kt

```diff
@@ -0,0 +1,5 @@
+package org.odk.collect.android.utilities
+
+enum class ScreenName {
```

Comment:
Sorry I should have said this in the last discussion, but this could probably go in `ApplicationConstants`.